src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -16,6 +16,7 @@
     .datagrid-host {
         display: flex;
         flex-flow: column nowrap;
+        padding-top: 0.5rem; //TODO: Remove. Temp fix. Matt to follow new UX recommendations with pinned columns
```

Comment:
Note: in the solution we discussed with @reddolan this is not needed. 
There is 0.5 rem above the datagrid itself and if the action-bar is present there is 1rem above that. 

I think this works until I get the pinned columns in there.

Reply:
Yep. Thats why I added the comment. When you rebase, you can just accept your changes.